lua/telescope/config.lua

```diff
@@ -37,6 +37,7 @@ function config.set_defaults(defaults)
   set("scroll_strategy", "cycle")
 
   set("layout_strategy", "horizontal")
+  set("layout_mirror_horizontal", false)
```

Comment:
We could have this part of layout_defaults:
```lua
layout_defaults = {
  horizontal = {
    width_padding = 0.1,
    height_padding = 0.1,
    preview_width = 0.6,
    mirror = false,
  },
  vertical = {
    width_padding = 0.05,
    height_padding = 1,
    preview_height = 0.5,
    mirror = true
  }
},

```

Reply:
And probably should

Reply:
That makes sense, I had figured that might be why you had the `layout_defaults` option as a TODO when I was deciding what to name the config option. Do you want me to implement allowing a config for `width_padding`, `height_padding`, and `preview_height` as well as well as adding the mirroring for the vertical layout?

Reply:
Oh wait they are just documented with TODO? Thats seems wrong. So they already exists. I just proposed to add mirror to that table.
You just need to add it to this table and then it should be available there.
https://github.com/nvim-telescope/telescope.nvim/blob/315e73698067ad96f71f1a13ee8b6c7567e61c34/lua/telescope/pickers/layout_strategies.lua#L64-L68

Like here `layout_config.mirror or default`
https://github.com/nvim-telescope/telescope.nvim/blob/315e73698067ad96f71f1a13ee8b6c7567e61c34/lua/telescope/pickers/layout_strategies.lua#L77

Reply:
![image](https://user-images.githubusercontent.com/37027883/109066689-54f71780-76e5-11eb-8e76-5f336c17a4f5.png)
Apologies, I misspoke. The `layout_defaults` option is marked as TODO in the README for telescope, not actually in the codebase itself (at least that I could find)

Reply:
@Conni2461 when you get a chance, take a look at the latest commit. I think I have fixed the confusion. I added `mirror` to the `layout_config` table and added a default `mirror` option of `false` to the `layout_defaults` table, for both `horizontal` and `vertical` layouts.